### ISSUE TITLE
FastConnect and Disconnect don't work on *first* run

### ISSUE THREAD

**Issue by ntc2** [derkomai/NordIndicator#11]
**The first time I run `NordIndicator.py`**, using `python3 NordIndicator.py install`, the sys tray icon successfully appears, and changes status (color and text) if I run `nordvpn [connect|disconnect]` from the command line, but **clicking on `FastConnect` and `Disconnect` in the indicator app's menu does *nothing***. The `Quit Nordindicator` option does quit the indicator.

However, **everything works as expected on subsequent runs** of `NordIndicator.py`, whether I start it by clicking on the `Applications -> Other -> NordVPN Indicator` launcher or by starting it fro the command line with `python3 NordIndicator.py`.

I don't see anything on stdout/stderr on this first run, and digging into this further I found a way to solve the problem:

``` diff
diff --git a/NordIndicator.py b/NordIndicator.py
index 9d49a73..e444c65 100644
--- a/NordIndicator.py
+++ b/NordIndicator.py
@@ -482,9 +482,7 @@ class InstallationHandler:
         self.createConfFile()
 
         # Launch
-        subprocess.Popen(['python3', f'{self.dstBinDir}/{self.scriptName}'],
-                         stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-
+        subprocess.Popen(['python3', f'{self.dstBinDir}/{self.scriptName}'])
 
     def createConfFile(self, extension=''):
```
This change results in the stdout/stderr of the subprocess going to the shell I ran `python3 NordIndicator.py install` in, and now everything works. Perhaps you used the `std*=subprocess.PIPE` stuff here on purpose, to avoid getting any output in the terminal where `install` was run, but I think it's better this way anyway, since if something goes wrong on the first run the user will see the output in their terminal.

**Comment by ntc2** [comment]
Forgot to add: I bet #9 was an instance of this, since the problem disappeared after the user restarted the indicator.

**Comment by dvilelaf** [comment]
I've just pushed a "fix" to both the issues (#11 & #12 ): now the installer won't auto-launch and the uninstaller won't auto-kill. You're right in that the implementations are buggy, so we'll let the user handle launching and closing.

**Comment by dvilelaf** [comment]
> Perhaps you used the std*=subprocess.PIPE stuff here on purpose, to avoid getting any output in the terminal where install was run, but I think it's better this way anyway, since if something goes wrong on the first run the user will see the output in their terminal.

The reason I used it was to let the installer process to end without blocking the console. If I remove PIPE as you propose, I cannot exit the installer without killing the child process.
